app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -60,7 +60,7 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
       label: yup.string().oneOf(CONTACTS.map(contact => contact.label)),
       value: yup.string().oneOf(CONTACTS.map(contact => contact.value)),
     }),
```

Comment:
Why is this not `required`?

Reply:
Yes, I had not included that as "required" since we were not sending tags and contact values from the form to the database. I was under the wrong impression and I realize that those frontend validations should be present even though in this case we are not sending any of these values to the backend. I have rectified this. It will also fix the mistake you pointed out in the loom video. Thanks for the review.